ota-plus-web/app/com/advancedtelematic/controllers/UserProfileController.scala

```diff
@@ -54,6 +54,8 @@ class UserProfileController @Inject()(val conf: Configuration,
     }
 
     fut.recover {
+      case UnexpectedResponse(u) if (u.status == TOO_MANY_REQUESTS) =>
+        TooManyRequests(u.body)
```

Comment:
We probably should log the body and give some different information to the JS application, like name of a service, if any. They can do nothing with the body and I would like to avoid showing it in UI by accident.

Reply:
Ok. But I am basically just doing what happens for an _Unauthorized_ error 3 lines down.

Reply:
Updated.